bundles/org.openhab.binding.easee/src/main/resources/OH-INF/thing/circuit-channel-groups.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="easee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-group-type id="circuit-settings">
+		<label>Circuit Settings</label>
+		<channels>
+			<channel id="maxCircuitCurrentP1" typeId="type-current">
+				<label>Max Circuit Current Phase 1</label>
+				<description>Max circuit current for phase 1.</description>
+			</channel>
+			<channel id="maxCircuitCurrentP2" typeId="type-current">
+				<label>Max Circuit Current Phase 2</label>
+				<description>Max circuit current for phase 2.</description>
+			</channel>
+			<channel id="maxCircuitCurrentP3" typeId="type-current">
+				<label>Max Circuit Current Phase 3</label>
+				<description>Max circuit current for phase 3.</description>
+			</channel>
+			<channel id="maxCurrents" typeId="rwtype-command">
+				<label>Max Circuit Currents Phase 1,2,3</label>
+				<description>Max circuit currents for phases 1,2,3.</description>
+				<properties>
+					<property name="writeCommand">SetMaxCircuitCurrents</property>
+					<property name="validationExpression">(0|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32);(0|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32);(0|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32)</property>
+				</properties>
+			</channel>
+			<channel id="offlineMaxCircuitCurrentP1" typeId="type-current">
+				<label>Offline Max Circuit Current Phase 1</label>
+				<description>Max circuit current for phase 1 in offline mode.</description>
+			</channel>
+			<channel id="offlineMaxCircuitCurrentP2" typeId="type-current">
+				<label>Offline Max Circuit Current Phase 2</label>
+				<description>Max circuit current for phase 2 in offline mode.</description>
+			</channel>
+			<channel id="offlineMaxCircuitCurrentP3" typeId="type-current">
+				<label>Offline Max Circuit Current Phase 3</label>
+				<description>Max circuit current for phase 3 in offline mode.</description>
+			</channel>
+			<channel id="offlineMaxCurrents" typeId="rwtype-command">
+				<label>Offline Max Circuit Currents Phase 1,2,3</label>
```

Comment:
Labels should be short (<25 chars, max 2-3 words). Can this be shortened? The same applies to other labels in this file.

Reply:
Unfortunately there are so many current types
- charger currents
-- max
-- dynamic
- circuit currents
-- max
-- dynamic
-- max offline

So it is hard to shorten without losing important information.

Reply:
I agree, it's not easee. :-) Perhaps this one could be shortened to something like "Offline Max Currents P1-3"? Description should help explain it better.

Reply:
I have shortened a lot of labels.